### PR TITLE
media-gfx/yafaray: bump to 3.5.1

### PR DESCRIPTION
**Pull request by waebbl**
Package-Manager: Portage-3.0.16, Repoman-3.0.2
Signed-off-by: Bernd Waibel <waebbl-gentoo@posteo.net>

### CODE REVIEW

File: media-gfx/yafaray/yafaray-3.5.1.ebuild [hunk 1]
```diff
@@ -0,0 +1,109 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{7,8,9} )
```
Comment (vaukai) [inline media-gfx/yafaray/yafaray-3.5.1.ebuild:6]: Isn't 3_7 already getting removed?

Comment (thesamesam) [reply media-gfx/yafaray/yafaray-3.5.1.ebuild:6]: For now, let's keep it, until it becomes a problem.

Comment (waebbl) [reply media-gfx/yafaray/yafaray-3.5.1.ebuild:6]: Usually they also get removed in one big push. If this were a new package or the python dep would be an addition to the previous version, I wouldn't add python-3.7.

File: media-gfx/yafaray/yafaray-3.5.1.ebuild [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+	opencv? ( media-libs/opencv:= )
+	openexr? ( media-libs/openexr:= )
+	png? ( media-libs/libpng:= )
+	python? ( ${PYTHON_DEPS} )
+	qt5? ( dev-qt/qtwidgets:5 )
+	tiff? ( media-libs/tiff )
+	truetype? ( media-libs/freetype:2 )
+"
+DEPEND="${RDEPEND}"
+BDEPEND="python? ( dev-lang/swig )"
```
Comment (juippis) [inline media-gfx/yafaray/yafaray-3.5.1.ebuild:44]: You might want to add `${PYTHON_DEPS}` here as well, inside `python? ( )`.

**Review by juippis — APPROVED** [review]
```
 * QA Notice: Files built without respecting CFLAGS have been detected
 *  Please include the following list of files in your report:
 * /usr/lib64/yafaray-plugins/libroughglass.so
 * /usr/lib64/yafaray-plugins/libieslight.so
 * /usr/lib64/yafaray-plugins/libtextureback.so
 * /usr/lib64/yafaray-plugins/libvolumetric.so
 * /usr/lib64/yafaray-plugins/libspherelight.so
 * /usr/lib64/yafaray-plugins/libSingleScatterIntegrator.so
 * /usr/lib64/yafaray-plugins/libsunlight.so
 * /usr/lib64/yafaray-plugins/libsunsky.so
 * /usr/lib64/yafaray-plugins/libblend.so
 * /usr/lib64/yafaray-plugins/libtgaHandler.so
 * /usr/lib64/yafaray-plugins/libbasicnodes.so
 * /usr/lib64/yafaray-plugins/libglass.so
 * /usr/lib64/yafaray-plugins/libNoiseVolume.so
 * /usr/lib64/yafaray-plugins/libarealight.so
 * /usr/lib64/yafaray-plugins/libSkyVolume.so
 * /usr/lib64/yafaray-plugins/libhdrHandler.so
 * /usr/lib64/yafaray-plugins/libEmptyVolumeIntegrator.so
 * /usr/lib64/yafaray-plugins/libshinydiffuse.so
 * /usr/lib64/yafaray-plugins/libsimple.so
 * /usr/lib64/yafaray-plugins/libequirectangularCamera.so
 * /usr/lib64/yafaray-plugins/libperspectiveCamera.so
 * /usr/lib64/yafaray-plugins/libspotlight.so
 * /usr/lib64/yafaray-plugins/libbidirpath.so
 * /usr/lib64/yafaray-plugins/libSunSky2.so
 * /usr/lib64/yafaray-plugins/libdirectional.so
 * /usr/lib64/yafaray-plugins/libSkyIntegrator.so
 * /usr/lib64/yafaray-plugins/libpathtrace.so
 * /usr/lib64/yafaray-plugins/libdirectlight.so
 * /usr/lib64/yafaray-plugins/liborthographicCamera.so
 * /usr/lib64/yafaray-plugins/libsppm.so
 * /usr/lib64/yafaray-plugins/libUniformVolume.so
 * /usr/lib64/yafaray-plugins/libgradientback.so
 * /usr/lib64/yafaray-plugins/libpointlight.so
 * /usr/lib64/yafaray-plugins/libEmissionIntegrator.so
 * /usr/lib64/yafaray-plugins/libbglight.so
 * /usr/lib64/yafaray-plugins/libcoatedglossy.so
 * /usr/lib64/yafaray-plugins/libglossy.so
 * /usr/lib64/yafaray-plugins/libExpDensityVolume.so
 * /usr/lib64/yafaray-plugins/libDebugIntegrator.so
 * /usr/lib64/yafaray-plugins/libbasictex.so
 * /usr/lib64/yafaray-plugins/libangularCamera.so
 * /usr/lib64/yafaray-plugins/libphotonmap.so
 * /usr/lib64/libyafaray_v3_plugin.so
 * /usr/lib64/libyafaray_v3_core.so
 * /usr/bin/yafaray-xml
```
You'll get a bug report about this.

File: media-gfx/yafaray/yafaray-3.5.1.ebuild (hunk 2 above)
Comment (waebbl) [reply media-gfx/yafaray/yafaray-3.5.1.ebuild:44]: Is it still needed if it's already in `RDEPEND`?

**Comment by waebbl** [comment]
> ```
>  * QA Notice: Files built without respecting CFLAGS have been detected
>  *  Please include the following list of files in your report:
> 
> You'll get a bug report about this.

Thanks! I just wonder why I missed those O.o

File: media-gfx/yafaray/yafaray-3.5.1.ebuild (hunk 2 above)
Comment (juippis) [reply media-gfx/yafaray/yafaray-3.5.1.ebuild:44]: Well I'd imagine python is still needed in the build-system, I see it goes to `DEPEND` but it usually gets put to `BDEPEND`.

Comment (waebbl) [reply media-gfx/yafaray/yafaray-3.5.1.ebuild:44]: Ok thanks, I'm gonna fix this together with the cflags issue.

Comment (juippis) [reply media-gfx/yafaray/yafaray-3.5.1.ebuild:44]: (I already did for ::gentoo repo) :)

Comment (waebbl) [reply media-gfx/yafaray/yafaray-3.5.1.ebuild:44]: Thanks a lot! :)

**Pull request closed without merging**